learn/cookbooks/computing_hugging_face_embeddings_gpu.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Computing Hugging Face embeddings with the GPU — Meilisearch documentation
+description: This guide for experienced users shows you how to compile a Meilisearch binary that generates Hugging Face embeddings with an Nvidia GPU.
+---
+
+This guide is aimed at experienced users working with a self-hosted Meilisearch instance. It shows you how to compile a Meilisearch binary that generates Hugging Face embeddings with an Nvidia GPU.
+
+## Prerequisites
+
+- A [CUDA-compatible Linux distribution](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#id12)
+- An Nvidia GPU with CUDA support
+- A modern Rust compiler
+
+## Install CUDA
+
+Follow Nvidia's [CUDA installation instructions](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html).
+
+## Verify your CUDA install
+
+After you have installed CUDA in your machine, run the following command in your command-line terminal:
+
+```sh
+nvcc --version | head -1
+```
+
+If CUDA is working correctly, you will see the following response:
+
+```
+nvcc: NVIDIA (R) Cuda compiler driver
```

Comment:
I think this one might be cut short, was it not?

Reply:
The code sample pipes the result into `head -1` so we only get the first line. As far as I understand, we only care about receiving a positive reply here stating cuda was correctly installed, so I thought using `head` would be simpler and more to the point.

I can change it if you think it's better to have the full output, though.

Reply:
OK I didn't see the head -1, that makes sense then

It doesn't need to be changed